### PR TITLE
Added braces to fix parsing errors

### PR DESCRIPTION
**Pull request by TheHollidayInn**
Turns out missing braces seemed to be causing the issue described here: https://github.com/HabitRPG/habitrpg/pull/4908 .

### CODE REVIEW

**Comment by Alys** [comment]
Missing braces - grrr! Languages that permit braces to be omitted should be harshly disciplined. I'll give this a test in the next day or two, unless someone else does first.

File: website/src/controllers/groups.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ var api = module.exports;
 var partyFields = api.partyFields = 'profile preferences stats achievements party backer contributor auth.timestamps items';
 var nameFields = 'profile.name';
 var challengeFields = '_id name';
-var guildPopulate = {path: 'members', select: nameFields, options: {limit: 15} };
+var guildPopulate = {path: 'members', select: nameFields, options: {limit: 2} };
```
Comment (gisikw) [inline website/src/controllers/groups.js]: Restricting this to two members means that only the guild leader and the current user show up.

screenshot 2015-04-02 06 07 17

Looks like the current user still shows up with this capped at 15 users. Is restricting the displayed group members desired behavior?

**Comment by TheHollidayInn** [comment]
Thanks, @gisikw! I was using the 2 to test out the functionality. 15 is correct. 

@Alys I agree :) Although, I am fond of Python.

**Comment by gisikw** [comment]
Ah, cool cool :)

**Comment by crookedneighbor** [comment]
@TheHollidayInn Looks like something got merged into develop that is causing a merge conflict. Would you mind pulling in the latest changes and sorting out the merge conflicts?

**Pull request closed without merging**